R/router.R

```diff
@@ -182,7 +182,9 @@ create_router_callback <- function(root, routes = NULL) {
 #' Creates router UI in Shiny applications.
 #'
 #' @param default Main route to which all invalid routes should redirect.
-#' @param ... All other routes defined with shiny.router::route function.
+#' @param ... <[`dynamic-dots`][rlang::dyn-dots]>
+#'   All other routes defined with shiny.router::route function.
+#'   It's possible to pass routes in dynamic way with dynamic dots
```

Comment:
Please remove the beginning (`<[`dynamic-dots`][rlang::dyn-dots]>`). Instead, after mentioning the possibility to use dynamic dots please:
- provide a link to `rlang` documentation with dynamic dots explanation
- mention the example

Reply:
Done, should I somehow add the link to the file in `examples/`